object-group-game/Inventory/Effect.cs

```diff
@@ -4,17 +4,11 @@
 
 namespace object_group_game
 {
-	class Effect
+	public class Effect
 	{
-		public string Status { get; private set; }
-		public string DisplayName { get; private set; }
-		public int Value { get; private set; }
-
-		Effect(string status, string displayName, int value)
-		{
-			Status = status;
-			DisplayName = displayName;
-			Value = value;
-		}
+		public int ID { get; set; }
+		public string StatusName { get; set; }
+		public string DisplayName { get; set; }
+		public int Value { get; set; }
```

Comment:
Bring back old changes here

Reply:
In this occasion, I was able to use existing class to describe table structure. All I needed was an additional property, which, I think, is not really an issue. Also, the constructor in redundant, because Effect class should only be received from the database.